File: new-algorithm/src/javaalgo/programmers/level1_평균_구하기.java
package javaalgo.programmers;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class level1_평균_구하기 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());
        int[] arr = new int[N];
        StringTokenizer st = new StringTokenizer(br.readLine());
        for (int i = 0; i < N; i++) {
            arr[i] = Integer.parseInt(st.nextToken());
        }

        System.out.println(solution(arr));
    }

    public static double solution(int[] arr) {
        int sum = 0;
        for (int i : arr) {
            sum += i;
        }
        return (double) sum / arr.length;
    }
}

File: new-algorithm/src/kotlinalgo/programmers/level1_평균_구하기.kt
package kotlinalgo.programmers

import java.io.BufferedReader
import java.io.InputStreamReader
import java.util.StringTokenizer

fun main() = with(BufferedReader(InputStreamReader(System.`in`))){
    val N = Integer.parseInt(readLine())
    val arr = IntArray(N)
    val st = StringTokenizer(readLine())
    for (i in 0 until  N) {
        arr[i] = Integer.parseInt(st.nextToken())
    }
    println(solution(arr))
}

fun solution(arr: IntArray): Double {
//    var sum = 0
//    for (i in arr) {
//        sum += i
//    }
//
//    return sum.toDouble() / arr.size.toDouble()
    return arr.average()
}

